src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -132,57 +215,179 @@ export default {
         });
       }).filter(Boolean)
     ),
+    handleKeyDown() {
```

Comment:
We have a mixing called keyboardNavigation that we use across the app to handle keyboard navigation 😃 Could we reuse it for this implementation too so we don't duplicate code?

Reply:
sure thanks for pointing that out, resolved in https://github.com/apple/swift-docc-render/pull/396/commits/aaa7e4f770e7fbb1c3daeb07fb0ea43867bc231f